### PR TITLE
Request mac address

### PR DESCRIPTION
**Pull request by bvanwijk**
The camera's now have a method to get the MAC address of the camera.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/camera/SimpleCameraFactory.java [hunk 1]
```diff
@@ -14,15 +14,17 @@
   public Camera createCamera(int index) throws InvalidCameraTypeException {
     Config config = ServerController.getInstance().getConfig();
     String address = config.getValue("camera_" + index + "_address");
-    if (address == null) {
+    String mACaddress = config.getValue("camera_" + index + "_macaddress");
+    if (address == null || mACaddress ==  null) {
       CameraController.logger.log("Can't create simplecamera"
           + " object with specified info of camera " + index,
           LogEvent.Type.CRITICAL);
       throw new InvalidCameraTypeException("The right information for"
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/camera/SimpleCameraFactory.java]: A space is missing here in the exception message.

File: backend/src/test/java/com/benine/backend/camera/ipcameracontrol/IpcameraTest.java [hunk 2]
```diff
@@ -39,6 +39,43 @@ public final void setUp() throws InvalidCameraTypeException{
...
   }
+  
+  @Test
+  public final void testGetMACAddress() throws CameraConnectionException {
+    parameterList = new ArrayList<Parameter>();
+    parameterList.add(new Parameter("FILE", "1"));
+
+    final HttpRequest request = HttpRequest.request("/cgi-bin/getinfo")
+                                    .withQueryStringParameters(parameterList);
+    mockServerClient.when(request).respond(HttpResponse.response().withBody("MAC=8C-C1-21-F0-46-C9 "
```
Comment (jochem725) [inline backend/src/test/java/com/benine/backend/camera/ipcameracontrol/IpcameraTest.java]: Maybe this message can be put in a separate `.txt` file to make the test more readable.

**Comment by jochem725** [comment]
I've added some small comments, but overall I agree with merging 😄

**Comment by egedebruin** [comment]
Looks good, I agree with merging.

**Comment by NaomidR** [comment]
Nice so see no findbug and checkstyle warnings and a high test coverage. Looks good, I agree with merging.

**Pull request merged**